controller-utils/pkg/kubernetes/kubernetes.go

```diff
@@ -532,7 +532,7 @@ func DecodeObjects(log logging.Logger, name string, data []byte) ([]*unstructure
 			if err == io.EOF {
 				break
 			}
-			log.Error(err, fmt.Sprintf("unable to decode resource %d of file %q", i, name))
+			log.Error(err, "unable to decode resource", "resourceIndex", i, "fileName", name)
```

Comment:
I'm not sure if we need constants for index and fileName?

Reply:
Oh, I think that makes sense. Especially file name.